platform-tests/src/acceptance/run_tests.sh

```diff
@@ -5,7 +5,7 @@ set -eu
 godep restore
 
 if [ -n "${GINKGO_FOCUS:-}" ]; then
-  go test -timeout 30m -ginkgo.focus "${GINKGO_FOCUS}"
+  ginkgo -p -nodes=16 -focus="${GINKGO_FOCUS}"
```

Comment:
Reasoning behind this is: 
* if this times out, custom acceptance tests fail, job fails, pipeline is blocked
* if this takes forever, job keeps running, pipeline is blocked

The main negative effect here is blockage of the pipeline, which is the same in both cases. However without timeout, there's chance that test might finish after some time & unblock pipeline. Moreover it leaves logs behind which might help analyze why it took so long.